packages/@lwc/engine-core/src/framework/html-element.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+// This is a temporary workaround to get the @lwc/engine-server to evaluate in node without having
+// to inject at runtime.
+export const HTMLElementConstructor: typeof HTMLElement =
+    typeof HTMLElement !== 'undefined' ? HTMLElement : (function () {} as any);
+export const HTMLElementPrototype = HTMLElementConstructor.prototype;
```

Comment:
Copied from the old `renderer.ts`. I didn't want to put this in `@lwc/renderer-abstract` because it's only needed in `@lwc/engine-core`.

Reply:
HTMLElement is already exposed by [the renderer](https://github.com/salesforce/lwc/pull/2598/files?diff=split&w=1#diff-97cddab3383c2dc463cad80d273b8fa9a878b325d923fb792423a0739e57d737R32-R37). It might be something we can refactor in a follow-up PR.